examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Startup.cs

```diff
@@ -22,6 +23,11 @@ public void ConfigureServices(IServiceCollection services)
 
             // We add a Settings model to the service container, which takes its values from the applications configuration.
             services.Configure<Settings>(Configuration.GetSection("WebDemo:Settings"));
```

Comment:
Should `EventHubConnection` be configured in the same way so it can be obtained via DI by `EventHubServcie`?

Reply:
Changed it to a DI.

---
In reply to: [464788619](https://github.com/Azure/AppConfiguration/pull/329#discussion_r464788619) [](ancestors = 464788619)